code/components/gta-net-rdr3/src/MumbleVoice.cpp

```diff
@@ -0,0 +1,770 @@
+#include "StdInc.h"

```

Comment:
I guess it will be better to share codebase between RDR3 and GTA5 for such stuff. You're re-using too much stuff anyway.

Reply:
But using ifdef for every native, pattern and registered native?
I think it will make a terrible chaos in this file

Reply:
Block along the lines of:

```cpp
#ifdef IS_RDR3
constexpr const auto patternBlah = "AA BB CC ? DD";
constexpr const auto nativeBlah = 0xDEADC0DEDEADC0DE;
#elif defined(GTA_FIVE)
constexpr const auto patternBlah = "AA BB AC ? DD";
constexpr const auto nativeBlah = 0xC0C0FEED;
#endif
```

would also work?

Reply:
You can also see `LoadStreamingFile.cpp`, it shares lots of code between games.

Reply:
gta-five-net MumbleVoice.cpp updated and symlink created